processor/k8sattributesprocessor/README.md

```diff
@@ -199,6 +201,51 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
+### Namespace-scoped RBAC
+When running the k8sattributesprocessor to receive telemetry traffic from pods in a specific namespace, you can use a k8s `Role` and `Rolebinding` to provide collector access to query pods and replicasets in the namespace. This would require setting the `filter::namespace` config as shown below.
+```yaml
+k8sattributes:
+  filter:
+    namespace: <WORKLOAD_NAMESPACE>
+```
+With the namespace filter set, the processor will only look up pods and replicasets in the selected namespace. Note that with just a role binding, the processor can not query metadata such as labels and annotations from k8s `nodes` and `namespaces` which are cluster-scoped objects.
```

Comment:
Should we also mention that `k8s.cluster.uid` cannot be detected with the namespace-scoped RBAC?

Reply:
updated with a note abt the cluster uid